tests/SignalSlot/RemoveTranslationSlotTest.php

```diff
@@ -0,0 +1,106 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace EzSystems\PlatformHttpCacheBundle\Tests\SignalSlot;
+
+use EzSystems\PlatformHttpCacheBundle\SignalSlot\RemoveTranslationSlot;
+use eZ\Publish\Core\SignalSlot\Signal\ContentService\RemoveTranslationSignal;
+use eZ\Publish\SPI\Persistence\Content\Location;
+use eZ\Publish\SPI\Persistence\Content\Location\Handler as LocationHandler;
+
+/**
+ * Test RemoveTranslationSlot for HttpCache.
+ */
+class RemoveTranslationSlotTest extends AbstractContentSlotTest
+{
+    protected $locationId = 43;
+    protected $parentLocationId = 45;
+
+    /**
+     * @var \eZ\Publish\SPI\Persistence\Content\Location\Handler|\PHPUnit_Framework_MockObject_MockObject
+     */
+    private $locationHandlerMock;
+
+    /**
+     * Check if required signal exists due to BC.
+     */
+    public static function setUpBeforeClass()
+    {
+        if (!class_exists(RemoveTranslationSignal::class)) {
+            self::markTestSkipped('RemoveTranslationSignal does not exist');
+        }
```

Comment:
> Oh, shouldn't we skip the test if class is not present @alongosz? otherwise it will fail on prev branches right?

Since this is unit test run directly from this package, composer install will always install newest kernel, so this shouldn't really happen. But just to be safe and consistent (we require ezpublish-kernel `^6.7`) I added this skip in 83c47b0.

Notes:
- done in `setUpBeforeClass` because data providers also rely on some usages of the Signal.
- I've checked on 6.7.x that if this class does not exist `class_exists(RemoveTranslationSignal::class)` returns false without warnings. This is ["documented" behavior](https://www.reddit.com/r/lolphp/comments/6fwu28/classnameclass_is_valid_for_any_identifier/) :laughing: 
Buf if you'd feel safer with string FQCN, I can fix that :)

ping @andrerom @Nattfarinn